kasparovsyncd/mqtt/transactions.go

```diff
@@ -83,18 +81,13 @@ func publishTransactionNotificationForAddress(transaction *apimodels.Transaction
 }
 
 // PublishAcceptedTransactionsNotifications publishes notification for each accepted transaction of the given chain-block
-func PublishAcceptedTransactionsNotifications(addedChainBlocks []*rpcclient.ChainBlock) error {
+func PublishAcceptedTransactionsNotifications(addedChainBlocks []rpcmodel.ChainBlock) error {
```

Comment:
Why not receive slice of pointers?
Ranging over slice of values is inefficient and error prone